receiver/filelogreceiver/README.md

```diff
@@ -19,35 +19,35 @@ Tails and parses logs from files.
+| `max_log_size`                      | `1MiB`                               | The maximum size of a log entry to read. A log entry will be truncated if it is larger than `max_log_size`. Protects against reading large amounts of data into memory.                                                                                         |
 | `max_concurrent_files`              | 1024                                 | The maximum number of log files from which logs will be read concurrently. If the number of files matched in the `include` pattern exceeds this number, then files will be processed in batches.                                                                |
 | `max_batches`                       | 0                                    | Only applicable when files must be batched in order to respect `max_concurrent_files`. This value limits the number of batches that will be processed during a single poll interval. A value of 0 indicates no limit.                                           |
-| `delete_after_read`                 | `false`                              | If `true`, each log file will be read and then immediately deleted. Requires that the `filelog.allowFileDeletion` feature gate is enabled.                                                                                                                      |
-| `attributes`                        | {}                                   | A map of `key: value` pairs to add to the entry's attributes                                                                                                                                                                                                    |
-| `resource`                          | {}                                   | A map of `key: value` pairs to add to the entry's resource                                                                                                                                                                                                      |
-| `operators`                         | []                                   | An array of [operators](../../pkg/stanza/docs/operators/README.md#what-operators-are-available). See below for more details                                                                                                                                     |
+| `delete_after_read`                 | `false`                              | If `true`, each log file will be read and then immediately deleted. Requires that the `filelog.allowFileDeletion` feature gate is enabled. Cannot use if `start_at` is set to "end".                                                                            |
```

Comment:
```suggestion
| `delete_after_read`                 | `false`                              | If `true`, each log file will be read and then immediately deleted. Requires that the `filelog.allowFileDeletion` feature gate is enabled. This option cannot be used if `start_at` is set to "end".                                                                            |
```

Reply:
Also, this option is ignored or will the code error if `start_at` is set to "end"?

Reply:
An exception will be raised. Should this be documented as well?

Reply:
yes, if that will crash teh collector, you can say something like: This option causes an error if it is used when `start_at` is set to "end".